src/lib/createWorkerBlobUrl.ts

```diff
@@ -1,3 +1,4 @@
+import isoworker from 'isoworker'
```

Comment:
Since you're using just `createContext`, maybe importing only that function is better. No need to bundle the workerization code.

Reply:
Hey! I tried this but I'm getting the following:

```
Can't import the named export 'createContext' from non EcmaScript module (only default export is available)
```

Reply:
Hey @101arrowz, just saw your comment in the other discussion. This is the message I'm getting when changing it to a named export, do you have any ideas?

Reply:
Hmm, it should be exporting ES6 modules. There is a `module` field of `package.json` and there is an ESM build exported. Maybe try importing from `fflate/browser` or `fflate/esm/browser.js`.

Reply:
Can't quite make it work with our current setup, could it be something related to our build config? @alewin

Reply:
@zant seems strange but I'll try to check it tomorrow and update you here!